src/stk/molecular/topology_graphs/host_guest/complex.py

```diff
@@ -185,9 +297,4 @@ def _get_scale(self, building_block_vertices):
         return 1
 
     def __repr__(self):
-        return (
-            f'host_guest.Complex('
-            f'guest_start={self._guest_start!r}, '
-            f'guest_target={self._guest_target!r}, '
-            f'displacement={self._displacement!r})'
-        )
+        return 'host_guest.Complex()'
```

Comment:
Can we fill this in a bit more, with the smiles of guests or something? Or would it be too noisy?

Reply:
I went with nothing because it was too noisy (large smiles strings are possible). I wish I could think of something nice here.

Reply:
Cool, np, fine by me.